docs/getting-started.md

```diff
@@ -195,6 +197,8 @@ development!) You can launch with the following command:
         -drive index=1,media=cdrom,file=$DISC \
         -usb -device usb-xbox-gamepad
 
+Note: If launching a pre-built binary from AppVeyor, replace `./i386-softmmu/qemu-system-i386` with `./xqemu`
+
```

Comment:
Here are some notes for what we should probably do in xqemu/xqemu to avoid this:

- We should have our own machine: bring back `qemu-system-xbox` (this was lost in QEMU2; and I strongly dislike breaking system-i386 which can be used to run things like xboxhdm)
- We should copy to the dist/ folder on all platforms, even if we might miss dependencies for now (can still be named `xqemu` as binary name)
- We should tell users to always run `./xqemu` from dist folder (stripped symbols; and contains all deps on Windows)
- We should tell developers to run `./xbox-softmmu/qemu-system-xbox` (contains symbols)

For now, this note is fine though.